Fixed the left mouse click for mesh editing not working with the new shortcut improvements. (#487)
Unity-Technologies/com.unity.probuilder: CHANGELOG.md

The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).

## [Unreleased]

### Fixed

> - Fixed a bug where it was no longer possible to edit meshes or enable view tools after the new shortcut improvements were added to the editor.

Reviewer: ```suggestion
- Fixed a bug that prevented editing meshes or enabling view tools after the new shortcut improvements were added to the Editor.
```

### Fixed

> - Fixed a bug where it was no longer possible to edit meshes or enable view tools after the new shortcut improvements were added to the editor.

Reviewer 2: This should be in a new ## [Unreleased] section and not in 5.2.0

Author: Oops! Fixed in [cb02fe5](https://github.com/Unity-Technologies/com.unity.probuilder/pull/487/commits/cb02fe5a2792e9639cd384b686adfef47d538d04)
